components/Modals/SignUp.tsx

```diff
@@ -99,16 +99,24 @@ export default function LoginModal({ onClose, state }) {
 
         <HorizontalScrollView>
           {[1, 2, 3, 4, 5].map((i) => (
-            <Avatar
-              key={i}
-              avatar={`/assets/avatars/${i}.png`}
-              size={80}
-              onClick={() => setAvatar(i)}
-              className={clsx('inline-block h-20 w-20 rounded-full border-2', {
-                'border-white': avatar === i,
-                'border-transparent': avatar !== i,
-              })}
-            />
+            <button
+              aria-label="select one out of the five avatar"
```

Comment:
I think that this is perhaps a bit redundant for the aria label, perhaps using the key `i` or even setting a name for each individual avatar

Reply:
Yes, this should state something like "Select avatar 1", "Select avatar 2"....

Imagine a blind user selects these buttons one after the other. Whenever a button is selected, they need to know what pressing it does. So each button needs to indicate which one will be selected on click.